highlight/highlight
# Fix comment modal esc

## Summary
Sometimes the comment modal does not fully unrender from the dom, leaving the modal overlay rendered and making the entire UI unclickable.

Return early when the modal is not rendered.

https://www.loom.com/share/1e09ae17d1444f8ca3362bc8bba01982?sid=364293d6-8afb-447d-8022-07d798de2f78

## How did you test this change?
1) View a session
2) Change to comment mode, and start leaving a comment
3) Press `esc` to exit the comment modal
- [ ] The UI should still be interactable

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
N/A
